File: Getting-Practice-with-Python-and-Javascript/1-First-Introductory-Steps/Javascript/Excercises/program-6.js
// 6-Crea un programa que pida al usuario una cantidad en grados Celsius y la convierta a grados Rankine. La fórmula para convertir de Celsius a Rankine es: R = (C + 273.15) * 9/5.



try {
    let userInput = parseFloat(prompt('Ingresa una cantidad en grados Celsius: '))
    if (isNaN(userInput)) {
        throw new Error('No es un numero valido. ')
    }
    let celsius = userInput
    let rankine = (celsius + 273.15) * 9 / 5
    console.log(`La temperatura ${celsius}° Celsius a Rankine es ${parseFloat(rankine)}`)
} catch (error) {
    console.error('Error: ', error.message);
} finally {
    console.log('Gracias!');
}

File: Getting-Practice-with-Python-and-Javascript/1-First-Introductory-Steps/Python/Excercises/program-6.py
# 6-Crea un programa que pida al usuario una cantidad en grados Celsius y la convierta a grados Rankine. La fórmula para convertir de Celsius a Rankine es: R = (C + 273.15) * 9/5.


try:
    userInput = float(input('Ingresa un numero en grados Celsius: '))
    Celsius = userInput
    rankine = (Celsius + 273.15) * 9/5
    print(f'La temperatura {Celsius}° Celsius a Rankine es {float(rankine)}')
except:
    print('Por favor, ingresa un numero valido. ')
